Tests/Microsoft.Teams.Apps.Tests/Activities/Invokes/FileConsentActivityTests.cs

```diff
@@ -0,0 +1,127 @@
+using Microsoft.Teams.Api;
+using Microsoft.Teams.Api.Activities.Invokes;
+using Microsoft.Teams.Api.Auth;
+using Microsoft.Teams.Apps.Activities.Invokes;
+using Microsoft.Teams.Apps.Annotations;
+using Microsoft.Teams.Apps.Testing.Plugins;
+
+namespace Microsoft.Teams.Apps.Tests.Activities.Invokes;
+
+public class FileConsentActivityTests
+{
+
+    private readonly App _app = new();
+    private readonly IToken _token = Globals.Token;
+    private readonly FileConsentController _controller = new();
+
+    public FileConsentActivityTests()
+    {
+        _app.AddPlugin(new TestPlugin());
+        _app.AddController(_controller);
+    }
+
+    //[Fact]
```

Comment:
Alex, these tests kept failing even though onFileConsent is valid extension method. do you have quick pointers on what might be wrong here? Otherwise, would remove this test for now and will pick up after current task

Reply:
@fallothakor in your controller handler your not calling `context.Next()` so the next handler will never be called